app/src/main/kotlin/com/wire/android/WireApplication.kt

```diff
@@ -116,7 +117,11 @@ class WireApplication : Application(), Configuration.Provider {
 
         val credentials = Credentials(clientToken, environmentName, appVariantName, applicationId)
         Datadog.initialize(this, credentials, configuration, TrackingConsent.GRANTED)
-        Datadog.setUserInfo(id = getDeviceId(this))
+        
+        if (android.os.Build.VERSION.SDK_INT >= 26) {
+            Datadog.setUserInfo(id = getDeviceId(this)?.sha256())
```

Comment:
we need to change getDeviceId to read the device id only for 26+
and the app is still reading the device id in other places  like `WireApplication`